graphics/chafa/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+github.setup        hpjansson chafa 1.4.1
+revision            0
+
+homepage            https://hpjansson.org/chafa
+
+description         Terminal graphics for the 21st century
+
+long_description    {*}${description}. Chafa is a command-line utility that \
+                    converts all kinds of images, including animated GIFs, \
+                    into sixel or ANSI/Unicode character output that can be \
+                    displayed in a terminal. It is highly configurable, with \
+                    support for alpha transparency and multiple color modes \
+                    and color spaces, combining selectable ranges of Unicode \
+                    characters to produce the desired output.
+
+categories          graphics
+license             LGPL-3
+platforms           darwin linux
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+checksums           rmd160  35a406ed8f4988caea800309cdd369cbe8325cf2 \
+                    sha256  f2a2a937a9687fc46a23c12b235aa847bf1c1e9d062c0d97f865b4b8504382ec \
+                    size    1212010
+
+depends_build-append \
+                    port:autoconf   \
+                    port:automake   \
+                    port:libtool    \
+                    port:pkgconfig
+
+depends_lib-append  port:glib2 \
+                    port:ImageMagick
+
+configure.cmd       ./autogen.sh
+use_configure       yes
+
+makefile.has_destdir yes
```

Comment:
Isn't passing `DESTDIR` the default even without the `makefile` PortGroup? In fact the Portfile works fine for me without the portgroup and this line.

Reply:
Thanks @neverpanic - I've removed `makefile.has_destdir`, but will keep the `makefile` portgroup.

Reply:
Does the port not build correctly without the makefile PortGroup, or what is the rationale for that? I've never seen any autoconf ports use that, so I'm a bit surprised to see it.

Reply:
Actually you're right - removed the portgroup as well.  Apologies for that @neverpanic